src/content/Sky.cpp

```diff
@@ -91,8 +90,9 @@ void Sky::interpolate(double deltaTime)
 //    if(inputGetKeyState(entry::Key::KeyO))
 //        deltaTime *= 10.0;
 
-    m_MasterTime += deltaTime;
-    m_MasterState.time = fmod(static_cast<float>(m_MasterTime / (60.0 * 60.0 * 24.0)), 1.0f);
+    m_MasterState.time += deltaTime / (60.0 * 60.0 * 24.0);
+    while (m_MasterState.time >= 1.0)
```

Comment:
Could use `m_MasterState.time = fmod(m_MasterState.time, 1.0f)` here I think. Tested it in Python and it worked, at least.